app/src/main/java/com/example/trashminder/presentation/newReminder/NewReminderFragment.kt

```diff
@@ -243,13 +239,30 @@ class NewReminderFragment : Fragment() {
         val datePickerDialog = DatePickerDialog(
             context,
             { _: DatePicker, mYear: Int, mMonth: Int, mDayOfMonth: Int ->
-                chosenDate.value = "$mDayOfMonth/${mMonth + 1}/$mYear"
+                var newDay = "$mDayOfMonth"
+                var newMonth = "${mMonth + 1}"
+                if (mDayOfMonth < 10) {
+                    newDay = "0$mDayOfMonth"
+                }
+                if (mMonth < 10) {
+                    newMonth = "0${mMonth + 1}"
```

Comment:
Why is the incrementation by 1 done only if `mMonth <10`?. Also, `mMonth` is a naming convention used in the older days of Java coding and it was applied only to global variables, which is not the case here. Please rename it to `month`. And all the other variables that use the same naming convention as well

Reply:
it's also done before the if, when newMonth is initialised. ( var newMonth = "${mMonth + 1}" )